helpers.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+func panicIfError(e error) {
```

Comment:
```
if err != nil {
  panic(err)
}
```

Is a standard golang idiom that is a standard way to check for error. I've never seen at the standard library that it moved to a separate function. It does not make much sense to do it in my opinion. So it is just better to inline the function.

Reply:
Came from other languages, always found it quite verbose and annoying, replace them back would be easy, I'll do it :)